### PR TITLE
nested the country_id under address in the user_sessions/new form to fix...

### PR DESCRIPTION
**Pull request by loucal**
I was getting an error when I clicked login and I had to do this to fix it.  Was I missing something?  Also, I wasn't getting the page at admin/overviews I was getting an error about recent_admin_users being nil and then the second time Ioaded it I was redirected to the homepage so I am never getting the login and password.  I will open up something separate once I get a chance to look into that one but let me know if you already have it figured out.

### CODE REVIEW

File: app/views/user_sessions/new.html.erb [hunk 1]
```diff
@@ -77,12 +77,13 @@
...
                   <div class="">
-                      <%= form.label :country_id, 'Country' %>
-                      <% if @user.errors[:country_id].first %>
-                        <span class='error-text'><%= @user.errors[:country_id].first %></span>
+                      <%= form.fields_for :address do |ff| %>
+                        <%= ff.label :country_id, 'Country' %>
+                        <% if @user.errors[:country_id].first %>
+                          <span class='error-text'><%= @user.errors[:country_id].first %></span>
+                        <% end %>
+                        <%= ff.select :country_id, select_countries, {:prompt => "Country"} %>
```
Comment (drhenner) [inline app/views/user_sessions/new.html.erb:85]: thanks for the feedback,  stripe_commerce originally came from a project that had country_id on the user model.  I'm just going to remove the country_id completely from this form.  I assume that works for you?

:question:

**Comment by drhenner** [comment]
I removed the country_id and this should be working... I'll close this issue at the end of the day unless you think I should be adding the country_id

File: app/views/user_sessions/new.html.erb (hunk 1 above)
Comment (loucal) [inline app/views/user_sessions/new.html.erb:85]: yeah, I don't think it makes sense for it to be there personally but it turned out there were more issues than that and I was never actually able to administer the site.  Have you ever tried to pull this project from scratch and set it up?  I hacked around a few errors but still couldn't get in to the admin backend and after an hour or two I had to give up and just configured and deployed spree instead.  If I remember correctly it had to do with the .admin? method (and consequently the is_admin filter etc.) returning false for my superuser...

Comment (drhenner) [inline app/views/user_sessions/new.html.erb:85]: I'll start from scratch tonight and get back to you.  It was working for a client but I did modify the open source version.

Comment (loucal) [inline app/views/user_sessions/new.html.erb:85]: Great, once you have the basics working I will help in any way I can.

**Comment by drhenner** [comment]
I tested things out... you should be good to go.  I had to adjust a couple things.  

Thanks

**Pull request closed without merging**